source/features/index.tsx

```diff
@@ -100,10 +100,7 @@ const globalReady: Promise<RGHOptions> = new Promise(async resolve => {
 
 	if (hotfixCSS.length > 0 || options.customCSS.trim().length > 0) {
 		await waitFor(() => document.head);
-		document.head.append(
-			<style>{hotfixCSS}</style>,
-			<style>{options.customCSS}</style>,
-		);
+		document.head.append(<style>{hotfixCSS}{options.customCSS}</style>);
```

Comment:
- Creating one element is theoretically faster than two
- This is safe because CSS parsing is incremental, which means whether the user's custom CSS is valid or not does not matter, as long as _our_ hotfix styles are

Reply:
This isn't safe actually:

```js
const hotfixCss = "body"
``` 

It's only safe because we're relatively careful about the hotfixes, hopefully, so it's ok.

Reply:
as long as our hotfix styles are> It's only safe because we're relatively careful about the hotfixes, hopefully, so it's ok. 

That's why I said _as long as our hotfix styles are_.

Consider it a reminder if there's something wrong with our hotfix 😉

Reply:
> since we don't have tests/lint for them

Yeah hotfixes are meant to be edited directly on GitHub.com, I don't really want to add CI. It wouldn't hurt to add the most basic validator for both the CSV and CSS files